# 小车uwb x,y 轴滤波后的效果
float32 car_cur_x
float32 car_cur_y
float32 car_cur_z
float32 car_yaw
float32 car_pitch
float32 car_roll

# 机器人uwb x,y 轴滤波后的效果
float32 robot_cur_x
float32 robot_cur_y
float32 robot_cur_z
float32 robot_yaw
float32 robot_pitch
float32 robot_roll


